src/mkeymap.rs

```diff
@@ -52,36 +53,17 @@ impl PartialEq<char> for KeyType {
 }
 
 impl<'b> MKeyMap<'b> {
-    pub fn new() -> Self {
-        MKeyMap::default()
-    }
     //TODO ::from(x), ::with_capacity(n) etc
     //? set theory ops?
 
-    #[deprecated(since = "3.0.0", note = "Use `contains` instead")]
```

Comment:
Why are we removing these?

Reply:
Because they aren't needed, like at all.